ui/app/adapters/allocation.js

```diff
@@ -6,12 +6,17 @@ import classic from 'ember-classic-decorator';
 export default class AllocationAdapter extends Watchable {
   stop = adapterAction('/stop');
 
-  restart(allocation, taskName) {
+  restart(allocation, taskName, allTasks) {
```

Comment:
I was going to suggest using `taskName` as the param you could optionally pass `"all"` to, to prevent you having to use a third parameter here, but I'm not sure if taskName can be arbitrary / one might call their task "all" in some scenario.

Either way: this seems like something worth breaking out into its own function like `restartAll()` rather than adding into the restart function.